netlify/landings/prism.yaml

```
@@ -24,14 +24,14 @@ hero:
 featureSection:
   actionBar:
     ctas:
-      - href: 'https://stoplight.io/p/docs/gh/stoplightio/prism/docs/getting-started/installation.md'
+      - href: 'https://stoplight.io/p/docs/gh/stoplightio/prism'
         title: Get Started with Prism
     enabled: true
     text: Do you want to mock your API in minutes?
 
   title: Work with your API before you write any code
   description: >-
-    Prism is an open source HTTP mock server that can mimic your API's behavior as if you already built it. Mock HTTP servers are generated from your OpenAPI v2/v3 (formerly known as Swagger) documents.
+    Prism is an open-source HTTP mock server that can mimic your API's behavior as if you already built it. Mock HTTP servers are generated from your OpenAPI v2/v3 (formerly known as Swagger) documents.
   features:
     - description: >-
         The waterfall model does not work well for building APIs. Learning at the very end what your
```

I think an 'a' is missing from this sentence.
```suggestion
        The waterfall model does not work well for building APIs. Learning at the very end what your potential consumers think about your software can be a source of significant problems. 
```